package.json

```diff
@@ -37,6 +37,11 @@
       }
     ],
     "commands": [
+      {
+        "command": "swift.createProject",
+        "title": "New Project",
```

Comment:
suggestion: "New" in title but "create" in ID, maybe make consistent?

suggestion: Typically have ellipsis suffix if the command will further prompt user for input

Reply:
Done.